src/components/CohortBuilder/Summary/index.js

```diff
@@ -2,20 +2,40 @@ import React from 'react';
 import styled from 'react-emotion';
 import Pie from 'chartkit/components/Pie';
 
-import { demographicPiesMock, topDiagnosesBarMock } from './mock';
+import { demographicPiesMock, topDiagnosesBarMock, studiesBarMock } from './mock';
 import Card from 'uikit/Card';
+import MultiHeader from 'uikit/Multicard/MultiHeader';
 import { CardWrapper } from 'uikit/Card/styles';
 import { Col, Row } from 'react-grid-system';
 
 import { withTheme } from 'emotion-theming';
 import HorizontalBar from 'chartkit/components/HorizontalBar';
 import { compose } from 'recompose';
 
-const participantTooltip = data => {
+const mostFrequentDiagnosisTooltip = data => {
   const participants = data.familyMembers + data.probands;
   return `${participants.toLocaleString()} Participant${participants > 1 ? 's' : ''}`;
 };
 
+const studiesToolTip = data => {
+  const { familyMembers, probands, name } = data;
+  const participants = familyMembers + probands;
+  return (
+    <div>
+      <div>{name}</div>
+      <div>{`${probands.toLocaleString()} Probands`}</div>
+      <div>{`${familyMembers.toLocaleString()} Family Members`}</div>
+      <div>{`${participants.toLocaleString()} Participant${participants > 1 ? 's' : ''}`}</div>
+    </div>
+  );
+};
+
+const sortDescParticipant = (a, b) => {
+  const aTotal = a.probands + a.familyMembers;
+  const bTotal = b.probands + b.familyMembers;
+  return aTotal <= bTotal ? -1 : 1;
+};
```

Comment:
@ciaranschutte I missed this in review way back, but just noticed that we're using `Array.sort` in the `HorizontalBar`. `Array.sort` actually does an in-place sort so, we prefer to use `lodash`'s version :P It takes a different callback signature though so will take some modifications to stuff currently using it. No need to worry about this now, just mentioning so we're aware ;)